src/test/scala/io/iohk/ethereum/blockchain/sync/FastSyncBlockBodiesRequestHandlerSpec.scala

```diff
@@ -83,6 +83,7 @@ class FastSyncBlockBodiesRequestHandlerSpec extends FlatSpec with Matchers {
     val fastSyncBlockBodiesRequestHandler: ActorRef =
       parent.childActorOf(SyncBlockBodiesRequestHandler.props(
         peer,
+        1.second,
```

Comment:
Minor detail: Should we maybe use the timeouts defined in `Timeouts.scala` and change it for `Timeouts.normalTimeout`?